src/Game/Game.cpp

```diff
@@ -19,7 +19,7 @@
 
 // C++ standard includes
 #include <sstream>
-#include <string>
+#include <string.h>
```

Comment:
http://www.qnx.com/developers/docs/6.5.0_sp1/index.jsp?topic=%2Fcom.qnx.doc.neutrino_lib_ref%2Fp%2Fputenv.html
It is needed only when you are not using "literal string". But.... our string is hardcoded and it is quite "literal" :)